sdk/servicebus/service-bus/review/service-bus.api.md

```diff
@@ -223,6 +223,31 @@ export interface PeekMessagesOptions extends OperationOptionsBase {
     fromSequenceNumber?: Long;
 }
 
+// @public
+export interface ProcessErrorContext {
+    entityPath: string;
+    errorSource: "complete"
+    /**
+     * Covers errors that occur when the user (or autoComplete) abandons a message.
```

Comment:
This is not for when the user abandons the message right? It is for when we abandon the message on behalf of the user if there was an error from the user callback or when the auto complete fails

Reply:
Changed the comment to reflect it. It's basically _always_ going to happen if the user throws so just needed to explain that.